web/src/lib/components/asset-viewer/photo-viewer.svelte

```diff
@@ -145,6 +146,20 @@
     return AssetMediaSize.Preview;
   });
 
+  $effect(() => {
+    if (assetFileUrl) {
+      void cast(assetFileUrl);
```

Comment:
Like this you basically lose any chance to provide the user with an helpful error. We have a `handleError` (or something like that) function in web that handles promises. I think that would be preferred here.